### DISCUSSION TITLE
Autocompletion followed by selection of environment end

### DISCUSSION THREAD

**Discussion by LoenderslootR** [WinEdt-Team/WinEdt-Support#82] [1.1 General Help and Discussion]
When editing LaTeX I use the autocompletion of environments, but only at the end, so typing \end{}, after which the autocompletion functionality adds the correct environment name between the brackets. Super handy.

However.... in WinEdt it then also selects the entire text line. Since I automatically continue typing, hence press <enter> to continue on a next line, the effect is that the selected text (\end{<env-name>}) disappeared. Super unhandy....

I could not find the setting to switch off the selection of the text line, while keeping the autocompletion. Can anyone help?

**Comment by Alex-Simonic** [comment]
Check [Discussion 54](https://github.com/WinEdt-Team/WinEdt-Support/discussions/54). This has been fixed and confirmed in the latest Build (20240613).
